### PR TITLE
fix(explore): include rest should not order by

### PR DESCRIPTION
**Pull request by skjindal93**
Include Rest only outputs a single aggregated result as Others. Having an order by without group by is not supported by the current OLAP store (Pinot)

### CODE REVIEW

**Review by samarth-gupta-traceable — APPROVED** [review]

File: gateway-service/src/main/resources/configs/common/application.conf [hunk 1]
```diff
@@ -38,7 +38,12 @@ interaction.config = [
   }
 ]
 
-timestamp.config = []
+timestamp.config = [
+  {
```
Comment (samarth-gupta-traceable) [inline gateway-service/src/main/resources/configs/common/application.conf:42]: is this change intentional ?

**Review by samarth-gupta-traceable — DISMISSED** [review]

File: gateway-service/src/main/resources/configs/common/application.conf (hunk 1 above)
Comment (skjindal93) [reply gateway-service/src/main/resources/configs/common/application.conf]: nice catch. thanks

File: gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/TheRestGroupRequestHandler.java [hunk 2]
```diff
@@ -80,14 +80,15 @@ private void mergeAndSort(
...
-    // Create a new request copied from the originalRequest but without any group by and set
+    // Create a new request copied from the originalRequest but without any group by, order by and set
     // includeRestGroup set to
     // false. This way we create a query with the same conditions as the original request.
     ExploreRequest.Builder requestBuilder =
         ExploreRequest.newBuilder(originalRequest)
             .clearGroupBy() // Remove groupBy
             .setIncludeRestGroup(false) // Set includeRestGroup to false.
             .setLimit(1) // Only one row
+            .clearOrderBy() // Remove orderBy
```
[the remark this answers is not in the document]
Comment (skjindal93) [reply gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/TheRestGroupRequestHandler.java]: Modified the existing set of tests, which takes care of this

File: gateway-service-impl/build.gradle.kts [hunk 3]
```diff
@@ -11,12 +11,11 @@ tasks.test {
 dependencies {
   api(project(":gateway-service-api"))
 
-  implementation("org.hypertrace.core.query.service:query-service-client:0.1.0")
-  implementation("org.hypertrace.core.attribute.service:attribute-service-client:0.1.0")
-  implementation("org.hypertrace.entity.service:entity-service-client:0.1.0")
-  implementation("org.hypertrace.entity.service:entity-service-api:0.1.0")
-  implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.1.3")
-  implementation("org.hypertrace.core.documentstore:document-store:0.1.0")
+  implementation("org.hypertrace.core.query.service:query-service-client:0.1.14")
```
Comment (tim-mwangi) [inline gateway-service-impl/build.gradle.kts:14]: Thanks!
